pkg/controller/atlasdatafederation/datafederation_controller.go

```diff
@@ -260,3 +263,31 @@ func (r *AtlasDataFederationReconciler) Delete(e event.DeleteEvent) error {
 
 	return nil
 }
+
+func managedByAtlas(ctx context.Context, atlasClient mongodbatlas.Client, projectID string, log *zap.SugaredLogger) customresource.AtlasChecker {
```

Comment:
The same project logic won't work here. linter is already complaining about the unused `projectID` param.